without explicit subsystem set, the linker tries to open a file
with the name of the comptibility version
(fatal error LNK1181: cannot open input file ",5.01")
diff -ur python3.org/PCbuild/_ctypes.vcxproj python3/PCbuild/_ctypes.vcxproj
--- python3.org/PCbuild/_ctypes.vcxproj	2014-05-19 19:06:01.274114800 +0200
+++ python3/PCbuild/_ctypes.vcxproj	2014-05-19 19:07:13.649079800 +0200
@@ -174,7 +174,7 @@
     </ClCompile>
     <Link>
       <AdditionalOptions>/EXPORT:DllGetClassObject,PRIVATE /EXPORT:DllCanUnloadNow,PRIVATE %(AdditionalOptions)</AdditionalOptions>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -187,7 +187,7 @@
     </ClCompile>
     <Link>
       <AdditionalOptions>/EXPORT:DllGetClassObject,PRIVATE /EXPORT:DllCanUnloadNow,PRIVATE %(AdditionalOptions)</AdditionalOptions>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -197,7 +197,7 @@
     </ClCompile>
     <Link>
       <AdditionalOptions>/EXPORT:DllGetClassObject,PRIVATE /EXPORT:DllCanUnloadNow,PRIVATE %(AdditionalOptions)</AdditionalOptions>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -210,7 +210,7 @@
     </ClCompile>
     <Link>
       <AdditionalOptions>/EXPORT:DllGetClassObject,PRIVATE /EXPORT:DllCanUnloadNow,PRIVATE %(AdditionalOptions)</AdditionalOptions>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
       <TargetMachine>MachineX64</TargetMachine>
     </Link>
@@ -221,7 +221,7 @@
     </ClCompile>
     <Link>
       <AdditionalOptions>/EXPORT:DllGetClassObject,PRIVATE /EXPORT:DllCanUnloadNow,PRIVATE %(AdditionalOptions)</AdditionalOptions>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -234,7 +234,7 @@
     </ClCompile>
     <Link>
       <AdditionalOptions>/EXPORT:DllGetClassObject,PRIVATE /EXPORT:DllCanUnloadNow,PRIVATE %(AdditionalOptions)</AdditionalOptions>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
       <TargetMachine>MachineX64</TargetMachine>
     </Link>
diff -ur python3.org/PCbuild/_decimal.vcxproj python3/PCbuild/_decimal.vcxproj
--- python3.org/PCbuild/_decimal.vcxproj	2014-05-19 19:06:01.274114800 +0200
+++ python3/PCbuild/_decimal.vcxproj	2014-05-19 19:07:13.649079800 +0200
@@ -176,7 +176,7 @@
       <AdditionalIncludeDirectories>..\Modules\_decimal;..\Modules\_decimal\libmpdec;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -189,7 +189,7 @@
       <AdditionalIncludeDirectories>..\Modules\_decimal;..\Modules\_decimal\libmpdec;..\Include;..\PC;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -199,7 +199,7 @@
       <AdditionalIncludeDirectories>..\Modules\_decimal;..\Modules\_decimal\libmpdec;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -212,7 +212,7 @@
       <AdditionalIncludeDirectories>..\Modules\_decimal;..\Modules\_decimal\libmpdec;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
       <TargetMachine>MachineX64</TargetMachine>
     </Link>
@@ -223,7 +223,7 @@
       <AdditionalIncludeDirectories>..\Modules\_decimal;..\Modules\_decimal\libmpdec;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -236,7 +236,7 @@
       <AdditionalIncludeDirectories>..\Modules\_decimal;..\Modules\_decimal\libmpdec;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </ClCompile>
     <Link>
-      <SubSystem>NotSet</SubSystem>
+      <SubSystem>Console</SubSystem>
       <BaseAddress>0x1D1A0000</BaseAddress>
       <TargetMachine>MachineX64</TargetMachine>
     </Link>
